### ISSUE TITLE
please make S3 storage optional

### ISSUE THREAD

**Issue by d2-d2** [ethanal/Nimbus#25]
Hello, you had this option in previous commits. What happened with that? Why did you revert it? Local storage would be a good alternative.

**Comment by Porco-Rosso** [comment]
I agree! I was about to deploy nimbus, and now I find this sadly missing!
